docs/synapse/devopsguide.rst

```diff
@@ -518,6 +518,33 @@ seen in the services ``cell.yaml`` file.
     ...
 
 
+.. _devops-task-nexustrim:
+
+Trimming the Nexus Log
+----------------------
+
+The Nexus log can be trimmed to reduce the storage size of any Cell that has Nexus logging enabled.
```

Comment:
war on "cell" language :D `any Synapse Service that has Nexus` plz :)

Reply:
updated